server/views/proposedAddress/rejected.pug

```diff
@@ -7,12 +7,21 @@ block content
   -var addressReview = data.curfew && data.curfew.curfewAddressReview || {}
   -var risk = data.risk && data.risk.riskManagement || {}
   -var savedData = data.proposedAddress && data.proposedAddress.rejected && data.proposedAddress.rejected.enterAlternative || {}
+  -var errorMessage = errorObject.enterAlternative
 
   div.pure-g.pure-u-1
     include ../includes/back
 
     h2.heading-large Curfew address rejected
     p The following address has been returned by the Responsible Officer:
+      if errorMessage
+        div.pure-g
+          div.error-summary.pure-u-1.pure-u-md-1-2(role="alert" aria-labelledby="error-summary-heading" tabindex="-1")
+            h2.heading-medium.error-summary-heading#error-summary-heading
+              | There is a problem
+            ul.error-summary-list
+                li
+                  a(href="#enterAlternativeYes")  #{errorMessage}
```

Comment:
there are (several for some reason?!) existing macros for managing errors - would it be possible to use one of those?

Reply:
Regarding the inline error message in alternativeForm.pug
There wasn't a mixin that applied the form-group-error class which paints a red vertical line on LHS of inputs. In fact in other modules, eg server/views/approval/release.pug,  the class is applied in a similar way to that used in the code above

Regarding error summary box in rejected.pug
The nearest fit was the server/views/includes/errorBannerWithDetailObject.pug mixin